sdk/subscription/azure-mgmt-subscription/CHANGELOG.md

```diff
@@ -1,147 +0,0 @@
-# Release History
-
-## 1.0.0 (2020-12-16)
-
-- GA release
-
-## 1.0.0b1 (2020-10-23)
```

Comment:
REstore changelog.

Reply:
Also when restoring changelog. If you intend to release before Jan 1st, put a warning that this will be the last version supporting py2.7.